.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Release
+
+on:
+  workflow_dispatch
+
+env:
+  OCAML_DEFAULT_VERSION: 5.0.0
```

Comment:
Nit: Apologies, I forgot about this earlier — it'd be safer to use 4.14 for this, given that this is what we test and bench on. Although it should not matter too much since this is just for building the release archive.